spec/spec_helper.rb

```diff
@@ -10,7 +12,9 @@
 
 require 'rspec'
 require 'facterdb'
+# rubocop:disable Style/MixinUsage
 include FacterDB
+# rubocop:enable Style/MixinUsage
```

Comment:
I think you can use RSpec to avoid it: https://www.rubydoc.info/github/rspec/rspec-core/RSpec/Core/Configuration#include-instance_method
```ruby
RSpec.configure do |config|
  config.include FacterDB
```

Reply:
Oh, nice. I didn't know about #include.

Reply:
fixed